submitTHORJob.py

```diff
@@ -119,10 +136,31 @@ def main():
             # Bucket already exists.
             pass
     bucket = gcs.bucket(args.bucket)
+
+    # Set up PubSub topic
+    if args.pubsub_topic is not None:
+        # Validate pubsub topic
+        split_topic = args.pubsub_topic.split("/")
+        if len(split_topic) != 4 or split_topic[0] != "projects" or split_topic[2] != "projects":
```

Comment:
Should the last one be 'topics' instead?

Reply:
Oops, yes. 

I should have tested this before pushing it up. Maybe there's a way to write a test of command-line tools, hmm hmm hmm... but I'll just fix it for now.